docs/FRIKA/about.md

```diff
@@ -0,0 +1,7 @@
+# About FRIKA
+
+FRIKA consists of three NVIDIA HGX Redstone GPU servers specifically dedicated to inferencing. The principal goal is to provide infrastructure for researchers and laboratories that want to offer their solutions (models/applications) as web services and thus promote their research/development work at UL FRI.
+
+FRIKA is currently running as a system of individual nodes that provide resources via Incus virtual machines and/or containers. Depending on requirements it is planned to be progressively further expanded. A possible transition to a k8s-based solution is also being actively investigated.
```

Comment:
A long term plan is to port all this to a Kubernetes based cluster.

Reply:
fixed.